connect/runtime/src/test/java/org/apache/kafka/connect/runtime/SampleSinkConnector.java

```diff
@@ -25,7 +25,7 @@
 
 public class SampleSinkConnector extends SinkConnector {
 
-    public static final String VERSION = "some great version";
+    public static final String VERSION = "latest";
```

Comment:
Plugins shouldn't return "latest" as their version, this is nonsense.

Maybe something that we should guard against.

Reply:
We'll have to test behaviors when plugins don't use traditional version numbers, stuff that cause DefaultArtifactVersion to "fallback" to just a qualifier.